### ISSUE TITLE
mantis setup issue

### ISSUE THREAD

**Issue by alxsimon** [PedroMTQ/mantis#43]
Hi,

I am running into an issue and unfortunately I can't pinpoint where the problem is as there is no useful error message in that case.

The `mantis setup` is ending before the end with an error.
Here is the last two lines of the log (I cut the full path):
```
Unpacking .../References/kofam/profiles.tar.gz to .../References/kofam/
Ran into an issue, check the log for details. Exitting!
```

Using Mantis v1.5.4 installed through bioconda

There is no obvious fault, space on disk is fine, memory too.
Let me know how I can try to help pinpointing the issue.
Thanks

**Comment by PedroMTQ** [comment]
Hello @alxsimon 
I will look into this over the weekend and will get back to you as soon as possible.

Regards,
Pedro

**Comment by PedroMTQ** [comment]
Hello @alxsimon 
I have tried to run this on my machine and the unpacking seems to be working fine, could you please try to run the setup again but just with the kofam database?

Please change your MANTIS.cfg file so that the other databases are deactivated, like so:
pfam_ref_folder=NA
nog_ref_folder=NA
tcdb_ref_folder=NA
ncbi_ref_folder=NA

Keep in mind the kofam servers have a quite low download speed, so this may take a while.

**Comment by alxsimon** [comment]
Thanks for checking @PedroMTQ

Seems the unpacking is now working (don't know what was going on).

But I am now running into another issue which I can pinpoint to the tcdb database:
```
Could not connect to url:                              
https://www.uniprot.org/uploadlists/                   
Ran into an issue, check the log for details. Exitting!
```

(Other databases seem to work on their own).

I don't think I have any network restrictions to access this website, it might be down on their side I don't know.
Tried the test function in this comment https://github.com/PedroMTQ/mantis/issues/24#issuecomment-863802707 which gave me this
```
<html>
<head><title>405 Not Allowed</title></head>
<body>
<center><h1>405 Not Allowed</h1></center>
<hr><center>nginx/1.21.6</center>
</body>
</html>
```

**Comment by PedroMTQ** [comment]
Hello @alxsimon 
It seems Uniprot changed their API so I had to update the calls Mantis makes.

In any case, this should be fixed now in version 1.5.5.

I'm in the process of updating the recipe in bioconda (1.5.4->1.5.5), and it should be available within 1 or 2 days.

Thanks!
Pedro

**Comment by alxsimon** [comment]
Oh right, sorry didn't notice the version mismatch between github and bioconda.
Thanks for the quick fix!
